python/lsst/obs/lsst/translators/imsim.py

```diff
@@ -98,3 +97,14 @@ def to_boresight_rotation_angle(self):
         angle = Angle(90.*u.deg) - Angle(self.quantity_from_card("ROTANGLE", u.deg))
         angle = angle.wrap_at("360d")
         return angle
+
+    @cache_translation
+    def to_physical_filter(self):
+        # Find throughputs version from imSim header data.  For DC2
+        # data, we used throughputs version 1.4.
+        for key, value in self._header.items():
+            if key.startswith('PKG') and value == "throughputs":
+                version_key = 'VER' + key[len('PKG'):]
+                throughputs_version = self._header[version_key].strip()
+                break
+        return '_'.join((self._header['FILTER'], f'sim_{throughputs_version}'))
```

Comment:
Maybe this instead?  Also, it might be nice to have a better error message (or default) if the above loop falls through.
```suggestion
        return '_'.join((self._header['FILTER'], 'sim', throughputs_version))
```

Reply:
yes, that's much better. thanks.

Reply:
Can you suggest an exception to raise if the throughputs version isn't found from the header data?

Reply:
Raising will prevent the file from being ingested. I think you probably want to return the value of the FILTER header and issue a log.warning (as we do in latiss.py).